packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -4080,47 +4314,203 @@ curl -v -X POST \
+
+```json
+{
+  "errorCode": "E0000068",
+  "errorSummary": "Invalid Passcode/Answer",
+  "errorLink": "E0000068",
+  "errorId": "oaei_IfXcpnTHit_YEKGInpFw",
+  "errorCauses": [
+    {
+      "errorSummary": "Your passcode doesn't match our records. Please try again."
+    }
+  ]
+}
+```
+
+###### Request Example for Verify TOTP Factor
+
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-d '{
+  "stateToken": "007ucIX7PATyn94hsHfOLVaXAmOBkKHWnOOLG43bsb",
+  "passCode": "657866"
+}' "https://${yourOktaDomain}/api/v1/authn/factors/ostfm3hPNYSOIOIVTQWY/verify"
+```
+
+###### Response Example for Verify TOTP Factor
+
+
+```json
+{
+  "expiresAt": "2015-11-03T10:15:57.000Z",
+  "status": "SUCCESS",
+  "relayState": "/myapp/some/deep/link/i/want/to/return/to",
+  "sessionToken": "00t6IUQiVbWpMLgtmwSjMFzqykb5QcaBNtveiWlGeM",
+  "_embedded": {
+    "user": {
+      "id": "00ub0oNGTSWTBKOLGLNR",
+      "passwordChanged": "2015-09-08T20:14:45.000Z",
+      "profile": {
+        "login": "dade.murphy@example.com",
+        "firstName": "Dade",
+        "lastName": "Murphy",
+        "locale": "en_US",
+        "timeZone": "America/Los_Angeles"
+      }
+    }
+  }
+}
+```
+
+#### Verify Push Factor
+
+
+<ApiOperation method="post" url="/api/v1/authn/factors/${factorId}/verify" />
+
+Sends an asynchronous push notification (challenge) to the device for the user to approve or reject.  The `factorResult` for the transaction will have a result of `WAITING`, `SUCCESS`, `REJECTED`, or `TIMEOUT`.
+
+##### Request Parameters for Verify Push Factor
+
+
+| Parameter      | Description                                          | Param Type | DataType | Required |
+| -------------- | ---------------------------------------------------  | ---------- | -------- | -------- |
+| factorId       | `id` of factor returned from enrollment              | URL        | String   | TRUE     |
+| stateToken     | [state token](#state-token) for current transaction  | Body       | String   | TRUE     |
+| rememberDevice | user's decision to remember device                   | URL        | Boolean  | FALSE    |
+| autoPush       | user's decision to send push to device automatically | URL        | Boolean  | FALSE    |
+
+**Okta Verify Push Details Pertaining to Auto-Push**
+
+* You don't need to pass the `autoPush` flag to Okta unless you have a custom sign-in flow that does not use the Okta sign-in widget, but want Okta to keep track of this preference.  The custom sign-in flow must still handle the logic to actually send the Auto-Push, since this param only deals with the Auto-Push setting.
+* If you pass the `autoPush` query param when verifying an Okta Verify Push factor, Okta saves this value as the user's preference to have the push notification sent automatically if the verification is successful (the user presses "Approve" on their phone).
```

Comment:
taps instead of presses and Approve bold rather than in quotes

....(the user taps **Approve** on their phone).